common/models.py

```diff
@@ -577,7 +577,7 @@ def serve(self, request):
             self.template = page_template
         return render(request, self.template, {
             'page': self,
-            'newsArticles': NewsArticle.objects.all().order_by('-date'),
+            'newsArticles': NewsArticle.objects.filter(live=True).order_by('-date'),
```

Comment:
This took me awhile to find...spent some time looking at[ templates/page/news.html](https://github.com/CenterForOpenScience/cos.io/blob/develop/templates/page/news.html#L31).. I was sure that I needed to just do something like: `news_item.live is True`. Since this was doing nothing, I spent awhile thinking that changes weren't showing up.  Then I spent awhile reading wagtail docs thinking this must be a setting somewhere, before just searching the codebase for "news"

Can some of these templates be deleted out of the admin app or at least marked that they are obsolete?

Reply:
Oh wow, good catch.

I don't think it's worth doing a full audit of the codebase at this time; for now, I think we can just clean litter up as we find it (Boy Scout Rule).

Btw, I've removed the unused template.

Reply:
great, thanks